content/network/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Network of Research Software Quality
+description: Network Information
+identifier: Network
+weight: 10
+showDate: false
+groupByYear: false
+---
+
+## Network of Research Software Quality ##
```

Comment:
This L2 header looks weird - it's the same name as the page itself. Maybe something like "Improving Research Software Quality".